cmake/modules/OmrDDRSupport.cmake

```diff
@@ -31,7 +31,8 @@ include(ExternalProject)
 # Make sure we have performed platform detection
 include(OmrPlatform)
 
-if(OMR_DDR AND CMAKE_CROSSCOMPILING)
+# DDR is may supported when cross compiling, unless we are on Windows with CYGWIN in which case we do have support
+if(OMR_DDR AND NOT OMR_OS_WINDOWS AND CMAKE_CROSSCOMPILING)
```

Comment:
@dnakamura could you confirm this will fix the problems seen in OpenJ9 OMR-Acceptance? I'm not 100% certain `OMR_OS_WINDOWS` is defined on CYGWIN, but I'd imagine it is.